libkbfs/folder_block_ops.go

```diff
@@ -941,17 +948,17 @@ func (fbo *folderBlockOps) newDirDataWithLBC(lState *lockState,
 				return block, true, nil
 			}
 
-			if rtype == blockReadParallel {
-				return nil, false, errors.New(
-					"LBC must be used with parallel reads")
+			lState := lState
```

Comment:
I know this is canonical, but because we potentially reassign it a few lines later, I'd prefer if the shadowed variable had a different name.